sympy/combinatorics/fp_groups.py

```diff
@@ -70,11 +71,41 @@ def __new__(cls, fr_grp, relators):
 
         obj._order = None
         obj._center = None
+
+        obj._rewriting_system = RewritingSystem(obj)
         return obj
 
     def _generators(self):
         return self.free_group.generators
 
+    def make_confluent(self):
+        '''
+        Try to make the group's rewriting system confluent
+
+        '''
+        self._rewriting_system.make_confluent()
+        return
+
+    def reduce(self, word):
+        '''
+        Return the reduced form of `word` in `self` according to the group's
+        rewriting system. If it's confluent, the reduced form is the unique normal
+        form of the word in the group.
+
+        '''
+        return self._rewriting_system.reduce(word)
+
+    def equals(self, word1, word2):
+        '''
+        Compare `word1` and `word2` for equality in the group
+        using the group's rewriting system. If the system is
+        confluent, the returned answer is necessarily correct.
+        (If it isn't, `False` could be returned in some cases
+        where in fact `word1 == word2`)
```

Comment:
Probably `None` should be returned instead of `False` if the system is not confluent.